src/main/scala/com/lambtors/poker_api/module/poker/domain/PlayerRepository.scala

```diff
@@ -0,0 +1,11 @@
+package com.lambtors.poker_api.module.poker.domain
+
+import scala.concurrent.Future
+
+import com.lambtors.poker_api.module.poker.domain.model.{GameId, Player, PlayerId}
+
+trait PlayerRepository {
+  def search(playerId: PlayerId): Future[Option[Player]]
+  def search(gameId: GameId): Future[List[Player]]
```

Comment:
I kind of think it makes more sense to have the Game own which players it has, because a player can be in more than one game, so maybe use the Game repository to find the players?

Reply:
do you think you can play poker in more than one game? :S